packages/gestalt/src/IconButton.css

```diff
@@ -6,10 +6,18 @@
   composes: block from "./Layout.css";
   composes: noBorder from "./Borders.css";
   composes: p0 from "./Whitespace.css";
-  composes: pointer from "./Cursor.css";
   background: transparent;
 }
 
+.enabled {
```

Comment:
Tiny nit: I'm not sure how the rest of Gestalt does it, but in past projects, I tended to include the default case (enabled, in this case) along with the base styles, and then use the alternate (disabled) state as overrides. Otherwise the component isn't truly "styled" until you add either an enabled or disabled style, and the code doesn't communicate this well.

Reply:
I tried pulling the pointer back into the base styles and only adding the disabled one when the prop is set but it doesnt work. The base styles always override this one. Im going to keep it as is since thats what Button.js does